### PR TITLE
23: Integrate PR code checks

### LINKED ISSUE #23: Integrate Github actions PR checks for PR
child of https://github.com/chennai-js/covid-info/issues/8

### PR DESCRIPTION
**Pull request by fasidOnGit**
Fixes #23

### CODE REVIEW

File: .github/workflows/pr_checks.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (saravanan10393) [inline .github/workflows/pr_checks.yml]: root branch is `main`

**Review by saravanan10393 — CHANGES_REQUESTED** [review]

File: .github/workflows/pr_checks.yml (hunk 1 above)
Comment (saravanan10393) [reply .github/workflows/pr_checks.yml]: also add this action for lint checking in the pipeline: https://github.com/marketplace/actions/eslint-plus

Comment (fasidOnGit) [reply .github/workflows/pr_checks.yml]: done

**Comment by fasidOnGit** [comment]
dont merge here. I am fixing package-lock issues

**Comment by fasidOnGit** [comment]
@saravanan10393 we are ready here

**Pull request merged**